src/client/components/collection/Item.js

```diff
@@ -21,6 +23,7 @@ const useStyles = makeStyles((theme) => ({
   cardDescriptionText: {
     '& p': {
       fontSize: 'large',
+      fontFamily: 'arial',
```

Comment:
This should probably be factored out, no? I think you can access it via `theme`.